src/Pusher.php

```diff
@@ -485,7 +486,10 @@ public function trigger($channels, $event, $data, $socket_id = null, $debug = fa
                 'error' => print_r($data, true),
             ), LogLevel::ERROR);
         }
-
+        if (PusherCrypto::is_encrypted_channel($channels[0])) {
+            $crypto = new PusherCrypto($this->settings['encryption_key']);
```

Comment:
I think we can just have a single instance of `PusherCrypto` which is initialised in the constructor.

Reply:
I guess it'll live as long as the user keeps the Pusher instance around, so sure